test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Culture.Tests.ps1

```diff
@@ -1,25 +1,64 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe "Get-Culture DRT Unit Tests" -Tags "CI" {
-    It "Should works proper with get-culture" {
-        $results = get-Culture
-        $results -is "System.Globalization.CultureInfo" | Should -BeTrue
-        $results[0].Name | Should -Be $PSCulture
-    }
-}
 
 Describe "Get-Culture" -Tags "CI" {
 
     It "Should return a type of CultureInfo for Get-Culture cmdlet" {
 
-	Get-Culture | Should -BeOfType CultureInfo
+        $culture = Get-Culture
+        $culture | Should -BeOfType [CultureInfo]
+        ($culture).LCID | Should -Be $host.CurrentCulture.LCID
+
+        Get-Culture -NoUserOverrides | Should -BeOfType [CultureInfo]
+    }
+
+    It "Should have `$PSCulture variable be equivalent to (Get-Culture).Name" {
+
+        (Get-Culture).Name | Should -Be $PsCulture
+    }
+
+    It "Should return the specified culture with '-Name' parameter" {
 
+        $ci = Get-Culture -Name ru-RU
+        $ci | Should -BeOfType [CultureInfo]
+        $ci.Name | Should -BeExactly "ru-RU"
+
+        $ci = Get-Culture -Name ru-RU -NoUserOverrides
+        $ci | Should -BeOfType [CultureInfo]
+        $ci.Name | Should -BeExactly "ru-RU"
+    }
+
+    It "Should return specified cultures with '-Name' parameter" {
+
+        $ciArray = Get-Culture "", "ru-RU"
```

Comment:
Seems like we should have a specific test case to handle invariant culture making it clear what is being validated

Reply:
Added the check and comment.